pom.xml

```diff
@@ -13,300 +13,314 @@
  *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
--->
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <groupId>com.ibm.eventstreams.connect</groupId>
-    <artifactId>kafka-connect-mq-source</artifactId>
-    <packaging>jar</packaging>
-    <version>1.3.5</version>
-    <name>kafka-connect-mq-source</name>
-    <organization>
-        <name>IBM Corporation</name>
-        <url>http://ibm.com</url>
-    </organization>
+ -->
```

Comment:
Also I think there is too much intentation change in this file could you please see if this can be changed to the previous intentation.